kelpie-f446/Core/UserDrivers/ThrusterDriver.c

```diff
@@ -2,17 +2,18 @@
  * ThrusterDriver.c
  *
  *  Created on: Oct 22, 2022
- *      Author: mingy
+ *      Author: mingy, eric
  */
 
 #include "ThrusterDriver.h"
 #include "SerialDebugDriver.h"
+#include "DataAggregationModule.h"
```

Comment:
Data aggregator should not be called in the driver layer. please move data aggregator stuff up to module layer